src/api.jl

```diff
@@ -444,3 +541,21 @@ function load_model(ctx::Context, database::AbstractString, engine::AbstractStri
     actions = [_make_load_model_action(name, model) for (name, model) in models]
     return _post(ctx, PATH_TRANSACTION; query = query(tx), body = body(tx, actions...), kw...)
 end
+
+
+
+# --- utils -------------------------
+# TODO: Delete this once https://github.com/JuliaWeb/HTTP.jl/issues/816 is addressed:
```

Comment:
explicitly note that this is copy/pasted from the lib

Reply:
done, thanks